server/routes/v1/index.ts

```diff
@@ -1,10 +1,12 @@
 import express from 'express';
 import userRouter from './user';
 import postRouter from './post';
+import commentsRouter from './comment';
 
 const routes = express.Router();
 
 routes.use('/users/', userRouter);
 routes.use('/posts/', postRouter);
+routes.use('/comments/', commentsRouter);
```

Comment:
```suggestion
routes.use('/posts/:postid/comments/', commentsRouter);
```

You can now access the post the comment is being created for by `postid`